core/src/main/scala/replpp/util/package.scala

```diff
@@ -14,5 +16,18 @@ package object util {
         } yield a +: acc
     }
   }
-  
+
+  def linesFromFile(path: Path): Seq[String] =
+    linesStreamFromFile(path).iterator.toSeq
+
+  def linesStreamFromFile(path: Path): IterableOnce[String] =
+    Files.lines(path).iterator().asScala
+
+  def deleteRecursively(path: Path): Unit = {
+    if (Files.isDirectory(path))
+      Files.list(path).forEach(deleteRecursively)
+
+    Files.deleteIfExists(path)
+  }
```

Comment:
```suggestion
  def deleteRecursively(path: Path): Unit = {
    Files.walk(path, maxDepth = 0).forEach(Files.deleteIfExists)
  }
```

Reply:
There's also the question of what do to with exceptions - do nothing like now, ignore exceptions, or rethrow but only after deleting everything that can be deleted. (But maybe it doesn't really matter with this only being used for scripts.)

Reply:
your suggested change has different semantics - the idea is to recursively delete the given file or directory. your suggestion doesn't go deep (maxDepth=0) and fails for nonEmpty directories

Reply:
Yes, I realized that a second after I pressed submit and ninja-edited the suggestion to drop the `maxDepth=0`. (Before that, I thought `maxDepth=0` would be no recursion limit. Then I checked again and realized it behaves like you say and there's an overload that doesn't require `maxDepth` at all.)